### PR TITLE
Chester refactor

### PR DESCRIPTION
**Pull request by Chesterwongz**
1. claimed `Money`, except for 1 function.
2. javadocs here and there

### CODE REVIEW

File: src/main/java/seedu/address/model/AddressBook.java [hunk 1]
```diff
@@ -104,12 +113,15 @@ public boolean hasClashingLesson(Lesson lesson, Lesson lessonToIgnore) {
...
+     *
+     * @param lessons The lessons we want to check.
+     * @return True if any of the given lessons clash with any existing lessons.
      */
     public boolean hasClashingLesson(Iterable<Lesson> lessons) {
         requireAllNonNull(lessons);
         return entries.hasClashes(lessons);
     }
-
+    //@@author
```
Comment (eeliana) [inline src/main/java/seedu/address/model/AddressBook.java:124]: Can help me add? Then add empty @@author before addPerson

```suggestion
    //@@author eeliana
```

Comment (eeliana) [reply src/main/java/seedu/address/model/AddressBook.java:124]: If you not claiming then no need

Comment (Chesterwongz) [reply src/main/java/seedu/address/model/AddressBook.java]: decided not to claim

**Comment by Chesterwongz** [comment]
merge #404 first

**Review by eeliana — APPROVED** [review]

File: src/main/java/seedu/address/model/lesson/CalendarEntryList.java [hunk 2]
```diff
@@ -375,7 +375,7 @@ public void updateUpcomingLessons() {
         entry.setTitle(entryTitle.toString());
         return entry;
     }
-
+    //@@author
```
Comment (angkoonhwee) [inline src/main/java/seedu/address/model/lesson/CalendarEntryList.java:378]: Sir, did you miss out this author name?

**Review by angkoonhwee — APPROVED** [review]

**Pull request merged**